src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -1749,7 +1748,7 @@ static void attn_reduce(T* dst, float* temp, size_t M, size_t S, size_t temp_str
 // N must be multiple of 16
 template <typename TDST,
           ov::element::Type_t SRC_PREC,
-          std::enable_if_t<(SRC_PREC != ov::element::u8 && SRC_PREC != ov::element::u4), bool> = true>
+          std::enable_if_t<none_of(SRC_PREC, ov::element::u8, ov::element::u4), bool> = true>
```

Comment:
can be converted to?:
```suggestion
          typename = std::enable_if_t<none_of(SRC_PREC, ov::element::u8, ov::element::u4)>>
```